chain/walker.go

```diff
@@ -90,6 +92,10 @@ func (c *Walker) WalkChain(ctx context.Context, node lens.API, ts *types.TipSet)
 			return xerrors.Errorf("get tipset: %w", err)
 		}
 
+		if int64(ts.Height()) < c.minHeight {
```

Comment:
Same change as in the last commit on https://github.com/filecoin-project/sentinel-visor/pull/640 and it has the same problem: the consensus task will not detect null rounds if the lower bound of the walk happens to be a null round. Will raise an issue to fix this.